src/test/java/kg/jarkyn/cobspecserver/doubles/ResponseControllerDouble.java

```diff
@@ -0,0 +1,30 @@
+package kg.jarkyn.cobspecserver.doubles;
+
+import kg.jarkyn.cobspecserver.*;
+
+import java.io.IOException;
+
+public class ResponseControllerDouble extends ResponseController {
+
+    private int timesResponded = 0;
+    private String response;
+
+    public ResponseControllerDouble(String response) {
+        super(null, null);
+        this.response = response;
+    }
+
+    @Override
+    public void respond(Client client) {
+        try {
+            timesResponded++;
+            client.getOutputStream().write(response.getBytes());
```

Comment:
i did wonder if I should, but I since this is a double i kept it explicit
also, it is not client's responsibility to write to itself... should it be?

I introduced StreamHandler later, I will just use it here